src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -0,0 +1,480 @@
+    if (!(dataPayload && dataPayload.data && dataPayload.data.length > 0)) {
+      return (
+        <PanelGridInfo numOfCols={eventView.numOfColumns()}>
+          <EmptyStateWarning>
+            <p>{t('No results found')}</p>
+          </EmptyStateWarning>
+        </PanelGridInfo>
+      );
+    }
+
+    const {meta} = dataPayload;
+    const fields = eventView.getFieldNames();
+    const lastRowIndex = dataPayload.data.length - 1;
+    const hasLinkField = eventView.hasAutolinkField();
+    const firstCellIndex = 0;
+    const lastCellIndex = fields.length - 1;
+
+    return dataPayload.data.map((row, rowIndex) => {
+      return (
+        <React.Fragment key={rowIndex}>
+          {fields.map((field, columnIndex) => {
+            const key = `${columnIndex}.${field}`;
+            const forceLinkField = !hasLinkField && columnIndex === 0;
+
+            const fieldRenderer = getFieldRenderer(field, meta, forceLinkField);
+            return (
+              <PanelItemCell
+                hideBottomBorder={rowIndex === lastRowIndex}
+                style={{
+                  paddingLeft: columnIndex === firstCellIndex ? space(1) : void 0,
+                  paddingRight: columnIndex === lastCellIndex ? space(1) : void 0,
+                }}
+                key={key}
+              >
+                {fieldRenderer(row, {organization, location})}
+              </PanelItemCell>
+            );
+          })}
+        </React.Fragment>
+      );
+    });
+  };
+
+  renderTable() {
+    const {isLoading, tableData: dataPayload} = this.props;
+    return (
+      <React.Fragment>
+        {this.renderHeader()}
+        {isLoading && (
+          <FloatingLoadingContainer
+            isLoading={true}
+            isReloading={isLoading && !!dataPayload}
+          />
+        )}
+        {this.renderContent()}
+      </React.Fragment>
+    );
+  }
+
+  renderError() {
+    const {error, eventView} = this.props;
+    return (
+      <React.Fragment>
+        <Alert type="error" icon="icon-circle-exclamation">
+          {error}
+        </Alert>
+        {this.renderHeader()}
+        <PanelGrid numOfCols={eventView.numOfColumns()}>{this.renderHeader()}</PanelGrid>
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {eventView, isLoading, error, tableData} = this.props;
+    const {hasFlagQueryBuilder, columnOrder, columnSortBy} = this.state;
+    const {
+      renderModalBodyWithForm,
+      renderModalFooter,
+    } = renderTableModalEditColumnFactory({
+      createColumn: this._createColumn,
+      updateColumn: this._updateColumn,
+    });
+
+    if (hasFlagQueryBuilder) {
+      return (
+        <GridEditable
+          isEditable={hasFlagQueryBuilder}
+          isLoading={isLoading}
+          error={error}
+          data={tableData ? tableData.data : []}
+          columnOrder={columnOrder}
+          columnSortBy={columnSortBy}
+          grid={{
+            renderHeaderCell: this._renderGridHeaderCell as any,
```

Comment:
This is generally not ideal to do 😬 . But this is fine for now.

Reply:
TS was complaining. I've spend some time trying to fix this but gonna bandaid over it for now.